examples/resources/qovery_job/resource.tf

```diff
@@ -0,0 +1,49 @@
+resource "qovery_job" "my_job" {
+  # Required
+  environment_id = qovery_environment.my_environment.id
+  name           = "test-job"
+
+  # Optional
+  state                = "RUNNING"
```

Comment:
We can remove the state, it isn't needed anymore since the `qovery_deployment` resource